Makefile

```diff
@@ -43,7 +43,5 @@ test: build
 	$(GOTEST) -v ./... 2>&1
 
 integration-test: build
-	cd packages/testdata && ../../$(BINARY_NAME) Gopkg.lock && cd -
```

Comment:
I think we still want this test no?? This validates that Gopkg.lock and dep support is still there.

Reply:
> I think we still want this test no?? This validates that Gopkg.lock and dep support is still there.

Great point, and this gets complicated too. The current processing wants a "real" project, and appears to take umbrage at the lock file not in a "go path"?

```
$ make integration-test
GO111MODULE=on CGO_ENABLED=0 go build -o nancy -v
cd packages/testdata && ../../nancy -p Gopkg.lock && cd -
Error: /Users/bhamail/sonatype/community/go/nancy/packages/testdata is not within a known GOPATH/src
For more information, check the log file at /Users/bhamail/.ossindex/nancy.combined.log
```

Any thought on how to mollify the tooling to make this test run?

Reply:
you either have to `go get github.com/sonatype-nexus-community/nancy` and work all your stuff from there (`~/go/src/....`) ......or you can fiddle your `GOPATH` and add where ever you have nancy living currently. CI will just work b/c it sets up GOPATH magic for you.

Reply:
I think I found a solution: set GOPATH to the current dir (after moving into the testdata dir). Seems to work locally too.  
d2e53d6